virt_lightning/shell.py

```diff
@@ -20,47 +22,75 @@
     "username": getpass.getuser(),
 }
 
+lang, encoding = locale.getdefaultlocale()
```

Comment:
Is there still people running an OS without UTF-8?

Reply:
Not, but people can use LC_ALL=C. I often see servers without UTF-8 files installed and connect to the servers with LC_ALL=C, because my main locale is ru_RU.UTF-8 and packages with russian language may not installed.